src/AddRecipe.js

```diff
@@ -0,0 +1,285 @@
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Indian
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("milk")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Italian
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("japanese")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Japanese
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("korean")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Korean
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("mediterranean")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Mediterranean
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("mexican")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Mexican
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("thai")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Thai
+                        </label>
+                    </div>
+                    <div className="form-check">
+                        <input className="form-check-input" type="checkbox" value="" id="defaultCheck1" onChange={() => {this.updateGenre("other")}}/>
+                        <label className="form-check-label" htmlFor="defaultCheck1">
+                            Other
+                        </label>
+                    </div>
```

Comment:
Super redundant - why not put your genres in an array and then loop through to create these?